modules/social_features/social_comment/social_comment.module

```diff
@@ -355,3 +356,14 @@ function social_comment_field_info_alter(&$info) {
     $info['comment']['list_class'] = '\Drupal\social_comment\SocialCommentFieldItemList';
   }
 }
+
+/**
+ * Implements hook_field_formatter_info_alter().
+ */
+function social_comment_field_formatter_info_alter(array &$info) {
+  /** @var \Drupal\Core\Extension\ModuleHandlerInterface $moduleHandler */
+  $moduleHandler = \Drupal::service('module_handler');
+  if (!$moduleHandler->moduleExists('social_comment_extra')) {
+    $info['comment_default']['class'] = SocialCommentDefaultFormatter::class;
```

Comment:
Do we want to override this for all comments if that module is enabled? Don't we rather want to update the comment formatter through the UI? This will allow others to override it if necessary. Or is that not working?